cscs-checks/apps/gromacs/gromacs_check.py

```diff
@@ -108,10 +108,10 @@ def __init__(self, **kwargs):
         self.tags |= {'production'}
         self.reference = {
             'dom:mc': {
-                'perf': (36.0, -0.15, None)
+                'perf': (38.36, -0.05, None)
             },
             'daint:mc': {
-                'perf': (73.0, -0.25, None)   # (82.0, -0.15, None)
+                'perf': (72.98, -0.50, None)
```

Comment:
The difference of `72.98` and `73.0` is negligible and it `73.0` is easier to parse. I would not change that. What do you think?